Destination/Backup.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Zenstruck\BackupBundle\Destination;
+use DateTime;
+
+/**
+ * Class Backup
+ *
+ * Abstraction of one backup file.
+ *
+ * @package Zenstruck\BackupBundle\Destination
+ */
+final class Backup
+{
+    /**
+     * @var string
+     */
+    private $key;
+
+    /**
+     * @var string
+     */
+    private $filename;
+
+    /**
+     * @var integer
+     */
+    private $size;
+
+    /**
+     * @var \DateTimeInterface
+     */
+    private $createdAt;
+
+    public function __construct($key, $filename, $size, $createdAt)
+    {
+        $this->key = $key;
+        $this->filename = $filename;
+        $this->size = $size;
+        $this->createdAt = is_integer($createdAt) ? date_timestamp_set(new DateTime(), $createdAt) : $createdAt;
+    }
+
+    /**
+     * @return string
+     */
+    public function getKey()
+    {
+        return $this->key;
+    }
+
+    /**
+     * @return string
+     */
+    public function getFilename()
+    {
+        return $this->filename;
+    }
+
+    /**
+     * @return int
+     */
+    public function getSize()
```

Comment:
Ok, fair point.

I still think we should use the 3rd party library for parsing the $size from config rather than a Util class.